ogre/src/OgreMaterial.cc

```diff
@@ -323,7 +329,8 @@ std::string OgreMaterial::NormalMap() const
 }
 
 //////////////////////////////////////////////////
-void OgreMaterial::SetNormalMap(const std::string &_name)
+void OgreMaterial::SetNormalMap(const std::string &_name,
+  const std::shared_ptr<common::Image>& /*_img*/)
```

Comment:
I didn't add the NormalMapData getter / copy the member variable since it seems normal maps are not implemented in OGRE1 anyway